tokio-async-await/Cargo.toml

```diff
@@ -20,11 +20,10 @@ categories = ["asynchronous"]
 async-await-preview = ["futures/nightly"]
 
 [dependencies]
-futures = "0.1.23"
```

Comment:
Specifying the patch release is intentional. The crate definitely does not work with futures version 0.1.0 ;)

Reply:
Uh... you're right. Fixed.